libraries/chain/include/eosio/chain/fork_database.hpp

```diff
@@ -80,6 +78,8 @@ namespace eosio::chain {
 
       void remove( const block_id_type& id );
 
+      bool in_valid_state() const; // not thread safe
```

Comment:
Seems safest to remove this method and just use `root()`.

Reply:
Thanks, removed that member.